src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,62 @@
+package duke;
```

Comment:
I like the way you organize your code in an OOP manner. The use of Command, Parser, Ui and Storage classes makes the logic of main method in your Duke class easy to understand.